### PR TITLE
Bump versions after automatic refresh fix

### PR DESCRIPTION
**Pull request by mateuszwojtczak**

### CODE REVIEW

File: packages/login_client/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 2.0.3
+- Try refreshing tokens on 401 responses
```
Comment (Albert221) [inline packages/login_client/CHANGELOG.md:2]: Add blank line before this one

```suggestion
- Try refreshing the token on `401 Forbidden` response
```

File: packages/login_client_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 2.0.1
+
+- Update login_client to 2.0.3
```
Comment (Albert221) [inline packages/login_client_flutter/CHANGELOG.md:3]:
```suggestion
- Upgrade login_client dependency to 2.0.3
```

File: packages/login_client_flutter/pubspec.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_secure_storage: ^4.2.0
-  login_client: ^2.0.1
+  login_client: ^2.0.3
```
Comment (Albert221) [inline packages/login_client_flutter/pubspec.yaml:17]: This won't pass tests as this version isn't yet published. Please move it to a separate Pull Request.

**Review by Albert221 — CHANGES_REQUESTED** [review]

File: packages/login_client_flutter/CHANGELOG.md (hunk 2 above)
Comment (mateuszwojtczak) [reply packages/login_client_flutter/CHANGELOG.md:3]: image

**Review by Albert221 — APPROVED** [review]
